Shared/ThresholdKeyView.swift

```diff
@@ -194,27 +182,17 @@ struct ThresholdKeyView: View {
                                 }
                                 // existing account
                                 else {
-                                    // check enough shares are available
-                                    if shareCount < threshold {
-                                        alertContent = "Not enough shares for reconstruction"
-                                        showAlert = true
-                                        resetAccount = true
-                                        showSpinner = SpinnerLocation.nowhere
-                                        return
-                                    }
                                     // import shares
-                                    // TODO: Handle failures of input_share gracefully. Its possible that locally available share was deleted.
                                     for item in shares {
-                                        guard let _ = try? await threshold_key.input_share(share: item, shareType: nil) else {
-                                            alertContent = "Incorrect share was used"
-                                            showAlert = true
-                                            resetAccount = true
-                                            showSpinner = SpinnerLocation.nowhere
-                                            return
+                                        do {
+                                            _ = try await threshold_key.input_share(share: item, shareType: nil)
+                                        } catch {
+                                            print("Incorrect share was used")
```

Comment:
Revert this, `print()` doesn't display anything to the user.

Reply:
reverted everything but return statement.
If the catch block contains a return, the next time we go around the loop, we will not be able to input the correct share and the for loop will end immediately. This is not the expected behavior, so we removed it.

```
alertContent = "Incorrect share was used, try importing another share.."
showAlert = true
resetAccount = true
showSpinner = SpinnerLocation.nowhere                                            
return // I deleted this

```

Reply:
If a share fails to be inserted it should break and show the error, not continue the loop.

Also the alertContent should be `"Incorrect share was used"`, since there is no other way to import a share.

Reply:
reverted